ivy_tests/test_ivy/test_functional/test_experimental/test_nn/test_activations.py

```diff
@@ -106,3 +106,34 @@ def test_prelu(
         x=x[0],
         slope=slope,
     )
+
+
+# relu
+@handle_test(
+    fn_tree="functional.ivy.experimental.relu6",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
I changed this to numeric since valid results in 'bool' datatype which is an invalid input, the result passes in all backends except jax where it always returns float64 instead of input datatype